VideoWeb/VideoWeb.UnitTests/Controllers/ParticipantController/UpdateJudgeDisplayNameTests.cs

```diff
@@ -13,9 +14,14 @@
 using VideoWeb.UnitTests.Builders;
 using EventHubEventType = VideoWeb.EventHub.Enums.EventType;
 using Autofac.Extras.Moq;
+using VideoApi.Contract.Enums;
 using VideoWeb.Contract.Request;
 using VideoWeb.Mappings;
 using VideoApi.Contract.Requests;
+using VideoApi.Contract.Responses;
+using VideoWeb.Contract.Responses;
+using VideoWeb.EventHub.Models;
+using LinkedParticipantResponse = VideoWeb.Contract.Responses.LinkedParticipantResponse;
```

Comment:
can we remove this, doesn't look like it is used

`using LinkedParticipantResponse = VideoWeb.Contract.Responses.LinkedParticipantResponse;`